src/providers/info.rs

```diff
@@ -68,6 +76,21 @@ impl InfoProvider for DefaultInfoProvider<'_> {
     }
 }
 
+impl DefaultInfoProvider<'_> {
+    pub fn exchange_usd_to(&self, currency_code: &str) -> Result<f64> {
+        let currency_code = currency_code.to_uppercase();
+        let url = format!("https://api.exchangeratesapi.io/latest?base=USD&symbols={}", &currency_code);
```

Comment:
Loading all exchange rates at once (e.g. no symbol param in the query to the exchange rate api) will allow a better caching